mail_full_expand/__manifest__.py

```diff
@@ -1,13 +1,12 @@
-# -*- coding: utf-8 -*-
-# © 2014-2015 Grupo ESOC <http://www.grupoesoc.es>
+# Copyright 2014-2015 Grupo ESOC <http://www.grupoesoc.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Mail full expand",
     "summary": "Expand mail in a big window",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Social Network",
-    "website": "http://www.grupoesoc.es/",
+    "website": "https://github.com/OCA/social",
     "author": "Grupo ESOC, Odoo Community Association (OCA)",
```

Comment:
You should switch here to `Tecnativa`, as Jairo moved from that company, or at least add us.